react/features/base/conference/middleware.any.ts

```diff
@@ -358,6 +391,11 @@ function _logJwtErrors(message: string, state: IReduxState) {
 
     message && logger.error(`JWT error: ${message}`);
     errorKeys.length && logger.error('JWT parsing error:', errorKeys);
+
+    dispatch(showErrorNotification({
```

Comment:
Feels a little weird to me in a function that would log only and that is named `_logJwtErrors` to actually show a notification. Maybe we should rename the function or move the notification out of it. WDYT?

Reply:
This is done. I also needed to force push to rebase and fix a conflict with master.